README.md

```diff
@@ -39,4 +39,5 @@ Findbugs Plugin version|Embedded SpotBugs/Findbugs version|Embedded Findsecbugs
 -----------------------|----------------------------------|----------------------------|---------------------------|--------------------|-----------------|------------------
 3.10                   | 3.1.11 (SpotBugs)                | 1.8.0                      | 7.4.3sb                   | 1.8|7.6~|5.10.1.16922
 3.11.0                 | 3.1.12 (SpotBugs)                | 1.8.0                      | 7.4.3sb                   | 1.8|7.6~|5.10.1.16922
-3.11.2-SNAPSHOT        | 3.1.12 (SpotBugs)                | 1.10.0                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
+3.11.2                 | 3.1.12 (SpotBugs)                | 1.10.0                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
```

Comment:
We have no `3.11.2`, please remove this line.

Reply:
@KengoTODA False assumption on my side, sorry. Fixed.